Assets/VRTK/Examples/042_CameraRig_MoveInPlace.unity

```diff
@@ -0,0 +1,2557 @@
+%YAML 1.1
```

Comment:
The `HeadsetCollisionFade` script should no longer be on the `Camera (eye)` object and should be put on the `CameraRig` object.

Also you can delete the `##Setup_Scene##` game object from the hierarchy (this is what is auto adding the Headset Collision Fade script as well).